pandas/tests/dtypes/test_dtypes.py

```diff
@@ -991,3 +991,8 @@ def test_is_dtype_no_warning(check):
 
     with tm.assert_produces_warning(None):
         check(data["A"])
+
+
+def test_check_dtype_empty_column():
```

Comment:
as a general rule, could you pls add an issue number below?

Reply:
Done